transformers/tests/modeling_unilm_test.py

```diff
@@ -0,0 +1,198 @@
+             sequence_labels, token_labels, choice_labels) = config_and_inputs
+            inputs_dict = {'input_ids': input_ids, 'token_type_ids': token_type_ids, 'attention_mask': input_mask}
+            return config, inputs_dict
+
+        def prepare_config_and_inputs_for_seq2seq_finetuning(self):
+            input_ids = ids_tensor([self.batch_size, self.seq_length], self.vocab_size)
+            masked_lm_labels = ids_tensor([self.batch_size, self.seq_length], self.vocab_size)
+            masked_pos = ids_tensor([self.batch_size, self.seq_length], self.seq_length)
+            masked_weights = ids_tensor([self.batch_size, self.seq_length], vocab_size=2)
+
+            input_mask = None
+            if self.use_input_mask:
+                input_mask = ids_tensor([self.batch_size, self.seq_length, self.seq_length], vocab_size=2)
+
+            token_type_ids = None
+            if self.use_token_type_ids:
+                token_type_ids = ids_tensor([self.batch_size, self.seq_length], self.type_vocab_size)
+
+            config = UnilmConfig(
+                vocab_size_or_config_json_file=self.vocab_size,
+                hidden_size=self.hidden_size,
+                num_hidden_layers=self.num_hidden_layers,
+                num_attention_heads=self.num_attention_heads,
+                intermediate_size=self.intermediate_size,
+                hidden_act=self.hidden_act,
+                hidden_dropout_prob=self.hidden_dropout_prob,
+                attention_probs_dropout_prob=self.attention_probs_dropout_prob,
+                max_position_embeddings=self.max_position_embeddings,
+                type_vocab_size=self.type_vocab_size,
+                is_decoder=False,
+                initializer_range=self.initializer_range)
+
+            return config, input_ids, token_type_ids, input_mask, masked_lm_labels, masked_pos, masked_weights
+
+        def create_and_check_unilm_model_for_seq2seq_finetuning(self, config, input_ids, token_type_ids, input_mask, masked_lm_labels, masked_pos, masked_weights):
+            model = UnilmForSeq2Seq(config=config)
+            model.to(torch_device)
+            model.eval()
+            loss = model(
+                input_ids, token_type_ids=token_type_ids, attention_mask=input_mask, 
+                masked_lm_labels=masked_lm_labels, masked_pos=masked_pos, masked_weights=masked_weights)
+
+            self.parent.assertListEqual(list(loss.size()), [])
+
+    def setUp(self):
+        self.model_tester = UnilmModelTest.UnilmModelTest(self)
+        self.config_tester = ConfigTester(self, config_class=UnilmConfig, hidden_size=37)
+
+    def test_config(self):
+        self.config_tester.run_common_tests()
+
+    def test_for_seq2seq_finetuning(self):
+        config_and_inputs = self.model_tester.prepare_config_and_inputs_for_seq2seq_finetuning()
+        self.model_tester.create_and_check_unilm_model_for_seq2seq_finetuning(*config_and_inputs)
+
```

Comment:
can you add an @slow  test that hardcodes the features you want your prettrained transformer to produce (ala roberta tests) so that we know whether it changes somehow?